website/docs/enterprise/getting-started/install-enterprise-single.mdx

```diff
@@ -0,0 +1,1165 @@
+:::
+
+Once the HTTP server starts, unauthenticated users will have to click 'Login With OIDC Provider' to log in.
+
+</TabItem>
+<TabItem value="Manual Cluster User" label="Manual Cluster User">
+
+:::danger Important
+This is an **insecure** method of accessing your dashboard. We only see useful in any of the following scenarios:
+- You cannot directly get started directly with OIDC so you create the cluster user for convenience.
+- Its a local and development environments.
+- You need to activate the user for emergency access to a damaged environment.
+
+For secure and fully-featured authentication we **strongly recommend** using an OIDC provider and remove the Cluster User
+before onboarding other users.
+:::
+
+**Configuring the Cluster User**
+
+Before you log in via the Cluster User account, you need to generate a bcrypt hash for your chosen password and store it as a secret in Kubernetes. There are several different ways to generate a bcrypt hash. This guide uses `gitops get bcrypt-hash` from our CLI.
+
+Generate the password by running:
+
+```bash
+PASSWORD="your password"
+echo -n $PASSWORD | gitops get bcrypt-hash
+
+$2a$10$OS5NJmPNEb13UgTOSKnMxOWlmS7mlxX77hv4yAiISvZ71Dc7IuN3q
+```
+
+Now create a Kubernetes secret to store your chosen username and the password hash:
+
+```bash
+kubectl create secret generic cluster-user-auth \
+  --namespace flux-system \
+  --from-literal=username=wego-admin \
+  --from-literal=password='$2a$10$OS5NJmPNEb13UTOSKngMxOWlmS7mlxX77hv4yAiISvZ71Dc7IuN3q'
+```
+
+You should now be able to login via the cluster user account using your chosen username and password.
+
+See how to harden this section in [Day 1 Cluster User](#cluster-user).
+
+</TabItem>
+</Tabs>
+
+### Authorization
+
+Weave GitOps Enterprise works with standard [Kubernetes RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)
+to define what actions can a subject do-over the existing resources.
+
+Weave GitOps requires authorization for the following subjects:
+
+- **Weave GitOps Users:** previously authenticated via either OIDC or Cluster User.
+- **Weave GitOps Service Account:** which talks to Kubernetes for user operations via [Impersonation](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#user-impersonation).
+- **Flux Service Accounts:** which talks to Kubernetes for deployments operations.
+
+When installing Weave GitOps Enterprise HelmRelease with the default values, a default authorization context for each of the
+previous subjects are created. This section introduces it, as well its motivation and how you could customise it.
+
+:::caution
+
+CLI does not support Authorization yet. It should be hadled as part of https://github.com/weaveworks/weave-gitops-enterprise/issues/3618
+
+
+// TODO cli should automate creating any manual step for example the bases patterns and to validate that the setup is stable.
+:::
+
+<Tabs groupId="authorization" default>
```

Comment:
Suggestion to simplify reading day0 is to move out also this informational stuff with the right linking to another part either apendix, sub-page ..